FlaskSite/utils/jwt_helper.py

```diff
@@ -0,0 +1,32 @@
+import jwt as pyjwt
+import datetime
+
+# Secret key for encoding and decoding the JWT token
+# move it to config.py 
+SECRET_KEY = "your_secret_key"
+
+
+def generate_token(user_id):
+    """
+    Generates a JWT token with the user_id as payload
+    """
+    payload = {
+        'user_id': user_id,
+        'exp': datetime.datetime.utcnow() + datetime.timedelta(hours=1),  # Token expiration
```

Comment:
datetime.datetime.utcnow() The method "utcnow" in class "datetime" is deprecated
You Could use it Like this : 
from datetime import datetime, timezone
datetime.now(timezone.utc)